config.json

```diff
@@ -1285,6 +1285,14 @@
           "control_flow_loops"
         ]
       },
+      {
+        "slug": "complex-numbers",
+        "name": "Complex Numbers",
+        "uuid": "ed6b5073-e3b1-4787-8551-ffffa9e7313d",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 6
```

Comment:
The core of the exercise isn't that difficult, but relying heavily on operator overloading makes this more challenging